src/components/cc-domain-management/cc-domain-management.smart.js

```diff
@@ -107,9 +107,8 @@ defineSmartComponent({
               'domainListState',
               /** @param {DomainManagementListStateLoaded} domainListState */
               (domainListState) => {
-                const hasPathPrefix = pathPrefix != null && pathPrefix !== '/';
                 // TODO: once the API returns actual ids, this should be adapted (either refetch or use the API response)
-                const id = hasPathPrefix ? hostname + pathPrefix : hostname;
```

Comment:
I cannot remember why I did that in the first place :/ I do remember that there were conflicts with some API routes not handling the trailing slash properly but it seems to work fine without it and usually I leave a comment to explain why I did something weird so it may be something I forgot to remove?

Reply:
I think the issue was as follows:
- when adding a new domain, I used to not send trailing slashes to the API because that's what was done before,
- so the `id` was the `hostname` without the path prefix if the path prefix was just "/",
- but at some point I decided to send trailing slashes to the API when adding a domain and I forgot to update the `id` here to reflect that.